src/contexts/Toast/ToastProvider.tsx

```diff
@@ -10,11 +10,17 @@ const setTimeout = window.setTimeout // To ensure return type is number
 interface ToastProviderValue {
   showToast: (...state: Partial<ToastProps>[]) => void
   updateDefaults: (state: Partial<ToastProps>) => void
+  clearAllToasts: () => void
+  toasts: ToastProps[]
 }
 
+type ToastState = ToastProps & { close: () => void }
+
 const ToastProviderContext = createContext<ToastProviderValue>({
   showToast: () => {},
   updateDefaults: () => {},
+  clearAllToasts: () => {},
+  toasts: [],
```

Comment:
So the reason why I poked around with the Toasts was so I could add a "clear all" method -- this gets invoked on login, so the residual "You have been logged out" notification gets cleared automatically.